application/groupview/InstanceDelegate.cpp

```diff
@@ -341,82 +342,4 @@ QSize ListViewDelegate::sizeHint(const QStyleOptionViewItem &option,
     return sz;
 }
 
-class NoReturnTextEdit: public QTextEdit
```

Comment:
Why is this removed?

Reply:
Its not used anymore, renaming is now handled via a dialog.